css/index.css

```diff
@@ -60,42 +102,62 @@ main {
     max-width: var(--max-content-width);
     margin: 0 auto;
     padding: var(--section-padding);
+    padding-top: var(--heading-height);
 }
 
-section h2{
-    font-size: var(--subheader-font-size);
-    color: var(--secondary-color);
-    margin-bottom: 1rem;
-    border-bottom: 2px solid var(--accent-color);
-    padding: 1rem;
-    padding-bottom: .5rem;
+/* ========== 4. Components ==========*/
+/* Navigation */
+nav {
+    display: flex;
+    flex-direction: row;
+    gap: space-between;
```

Comment:
gap property can only have numeric values like 30px (it will specify the space between the items in the flex container), it should also be applied directly in the flex container (in this case nav ul, since the `<li>` are the flex items.

Reply:
If you use the gap example -> gap: 30px; in the nav, the only children that nav has is the ul item, so it doesn't have the expected result, likely you want to have that space for the list items, similar when applying the flex properties, they need to be applied directly in the list items direct parent (in this case nav ul).